.github/workflows/test.yml

```diff
@@ -61,11 +61,17 @@ jobs:
           source variables.sh
           git clone --depth=1 https://github.com/vauxoo/maintainer-quality-tools.git -b master ${HOME}/maintainer-quality-tools
           export PATH=${HOME}/maintainer-quality-tools/travis:${HOME}/gitlab_tools:${PATH}
+          export TRAVIS_PYTHON_VERSION="3.7"
           source ${REPO_REQUIREMENTS}/virtualenv/python${TRAVIS_PYTHON_VERSION}/bin/activate
           source ${REPO_REQUIREMENTS}/virtualenv/nodejs/bin/activate
           source /usr/local/rvm/scripts/rvm
           cp test-oca_dependencies.txt oca_dependencies.txt
           travis_install_nightly
+          wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+          apt update
+          apt install fonts-liberation
+          apt install libgbm1
+          dpkg -i google-chrome-stable_current_amd64.deb
```

Comment:
This is already done
[here](https://github.com/Vauxoo/maintainer-quality-tools/blob/8a82463a0d8bff888272f7e619b47796c3b17db4/travis/travis_install_nightly#L153)
by the command run just above (`travis_install_nightly`), so it shouldn't be required.

Reply:
After you removed this, I noticed the issue is the latest version of Chrome can't be installed on this Ubuntu version.

After some tests, I figured out the latest Chrome version that may run under Ubuntu 14.04 is 107.0.5304.110.

Adding the following to the variables.sh file should fix the problem:

`export CHROME_VERSION="107.0.5304.110-1"`